.editorconfig

```diff
@@ -0,0 +1,10 @@
+# top-most EditorConfig file
+root = true
+
+[*.{rb,css,js,jsx,erb,slim,scss}]
```

Comment:
not yaml/yml?

Reply:
iirc some yaml files have weird significant whitespace requirements, like `.circleci/config.yml`, so i left them out